test/slow-save.js

```diff
@@ -0,0 +1,113 @@
+const fs = require('fs')
+const validate = require('ssb-validate')
+const ssbKeys = require('ssb-keys')
+const path = require('path')
+const push = require('push-stream')
+const { prepareAndRunTest, addMsg, helpers } = require('./common')()
+const rimraf = require('rimraf')
+const mkdirp = require('mkdirp')
+
+const dir = '/tmp/jitdb-slow-save'
+rimraf.sync(dir)
+mkdirp.sync(dir)
+
+var keys = ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
+
+prepareAndRunTest('wip-index-save', dir, (t, db, raf) => {
```

Comment:
I guess we can remove the wip from the names here?

Reply:
Just a minor thing, the PR overall looks good.

Reply:
Oh, I'm slow today. Scratch this.

Reply:
Yeah it looks a bit weird as a test name, I also got a tiny bit confused sometimes